samples/bluetooth/mesh/light/sample.yaml

```diff
@@ -24,3 +39,12 @@ tests:
     platform_allow: nrf52840dk_nrf52840 nrf21540dk_nrf52840
     extra_args: OVERLAY_CONFIG=overlay-dfu.conf
     tags: bluetooth ci_build
+  sample.bluetooth.mesh.light.dfu.sysbuild:
```

Comment:
@alxelax could you take a look at this? I believe this build target will run into the known issue you reported

Reply:
Yep. Seems this is not sufficient. If to build sample with `--sysbuild` option and the overlay it doesn't add mcuboot to the final image.
Probably it might have been fixed but this is the known issue on rc1.

Reply:
sure

Reply:
Most applications will fail to build until the hwmv2/sysbuild in ncs upmerge is merged